R/ALG_BPR.R

```diff
@@ -35,7 +35,7 @@ BPR <- function(data, k = 10, lambda = 0.05, regU = 0.0025, regI = 0.0025, regJ
     
     while (!isConverged(x, p)) {
         
-        for (s in 1:100 * row_x) {
+        for (s in 1:(100 * row_x)) {
```

Comment:
This python implementation calculates the updates the same way that I proposed: [https://github.com/gamboviol/bpr/blob/master/bpr.py#L80-L98](https://github.com/gamboviol/bpr/blob/master/bpr.py#L80-L98). `z = 1.0/(1.0+exp(x))`.